cmd/machine-config-daemon/pivot.go

```diff
@@ -228,17 +233,18 @@ func pullAndRebase(container string) (imgid string, changed bool) {
 
 	// If we're passed a non-canonical image, resolve it to its sha256 now
 	isCanonicalForm := true
-	if _, err := daemon.GetRefDigest(container); err != nil {
+	if _, err = daemon.GetRefDigest(container); err != nil {
```

Comment:
leave `:`as it's scoped to this if (w/o changing below as well)

Reply:
I don't think that works because we need to set the named return value, right?